src/Components/Checkboxes/Checkboxes.js

```diff
@@ -0,0 +1,27 @@
+export const Checkboxes = (props) => {
+  const {
+    options,
+    label,
+    handleChange
+  } = props
+
+  return (
+    <>
+      <p htmlFor={label}>Select Insurance Providers</p>
+
+        {options.map(item => (
+          <div key={item}>
+            <input
+              type='checkbox'
+              id={item}
+              name={item}
+              value={item}
+
+              onChange={handleChange}
+            />
+            <label htmlFor={item}>{item}</label>
```

Comment:
I like the way this is set up to cut down on repeated code, and potentially being more dynamic